docs/charging.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: charging
+id: payments
```

Comment:
if this `id` is changing, should the file name?

Reply:
👍